Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -463,67 +378,27 @@
   },
   {
     "Input": "Estare afuera el tercer cuatrimestre de 2016",
+    "NotSupported": "dotnet",
+    "NotSupportedByDesign": "javascript, python, java",
     "Results": [
       {
-        "Text": "el tercer cuatrimestre de 2016",
```

Comment:
In English, this test resolves `the third quarter of 2016` from the input `I'll be out the third quarter of 2016` maintaining `the` keyword, so in Spanish, we should keep `el` equivalent.